instance/tests/management/test_instance_statistics_csv.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2019 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance - Logs Activity CSV unit tests
+"""
+# Imports #####################################################################
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.utils.six import StringIO
+from django.test import TestCase
+
+
+# Tests #######################################################################
+
+class InstanceStatisticsCSVTestCase(TestCase):
+    """
+    Test cases for the `activity_csv` management command.
+    """
+    def test_no_qualified_domain(self):
+        """
+        Verify that the command correctly notifies the user that there are no active app servers.
+        """
+        with self.assertRaises(CommandError) as context_manager:
+            call_command(
+                'instance_statistics_csv'
+            )
+            self.assertIn('Error: the following arguments are required', str(context_manager.exception))
+
+    def test_no_matching_instances(self):
+        """
+        Verify that the command correctly notifies the user that there are no active app servers.
```

Comment:
@pcockwell, `no matching instances` is not the same as the `no active appservers`. Can you write a separate test for the latter?

Reply:
Resolved in next commit by adding a few new tests